tfhe/src/shortint/ciphertext/common.rs

```diff
@@ -0,0 +1,362 @@
+use super::super::CheckError;
+pub use crate::core_crypto::commons::parameters::PBSOrder;
+use crate::shortint::parameters::{CarryModulus, MessageModulus};
+use serde::{Deserialize, Serialize};
+use std::cmp;
+use std::fmt::Debug;
```

Comment:
could be in the mod.rs itself ?

Reply:
I prefer it this way
I think it's clearer to have the mod only containing other modules (and possibly reexports)

Reply:
ok